traffic_control/models.py

```diff
@@ -160,3 +176,101 @@ class Meta:
 
     def __str__(self):
         return "%s %s %s" % (self.id, self.code, self.value)
+
+
+class TrafficSignReal(models.Model):
```

Comment:
This model represents a regular traffic sign that's physically somewhere, right? In that case I don't really see the point of having `Real` suffix in the model name. In my opinion `TrafficSign` is enough in the context of this project and that the `Real` doesn't have additional value.

Reply:
In this application context, it is important to separate plan and real models from each other. `Real` makes the model more explicit, thus I would prefer leaving it there.

Reply:
True. Explicit is better than implicit. Let's leave this like this. 👍